docs/user-guide/cli-install-cli-next.md

```diff
@@ -60,7 +55,8 @@ Install the Zowe CLI `@next` version from the online npm registry or download it
 You can now try out the [technical preview features](cli-development-roadmap-next.md).
```

Comment:
Perhaps steps 3,4,5,6 here and steps 7,8,9 in the section below could be modified to instead use the `zowe config convert-profiles` migration utility. Perhaps these steps could be consolidated in a shared section on migration as it is the same content being maintained in two different places.

Perhaps @t1m0thyj has some doc on this migration that could be placed/linked here.

Reply:
The steps mentioned above could be replaced with something like the following:

3. Issue the command below to migrate your CLI profiles from v1 to v2. Old profile data is backed up in case you want to revert to Zowe v1 LTS in the future.
  `zowe config convert-profiles`
4. (Optional) If you are confident you want to completely remove your v1 profile data, issue the command below. It is recommended not to do so until you have tested v2 for a while.
  `zowe config convert-profiles --delete`